crucible/src/Lang/Crucible/CFG/Reg.hs

```diff
@@ -1007,7 +1013,16 @@ instance PrettyExt ext => Pretty (CFG ext s init ret) where
          , vcat (pretty <$> cfgBlocks g) ]
 
 ------------------------------------------------------------------------
--- SomeCFG
+-- SomeCFG, AnyCFG
 
 -- | 'SomeCFG' is a CFG with an arbitrary parameter 's'.
 data SomeCFG ext init ret = forall s . SomeCFG !(CFG ext s init ret)
+
+-- | Control flow graph.  This data type closes existentially
+--   over all the type parameters.
```

Comment:
...all of the type parameters except for `ext`.

Reply:
Fixed, and fixed the corresponding comment in `CFG.Core`